### ISSUE TITLE
Mob Masher causes fps to drop drasticly

### ISSUE THREAD

**Issue by Infectedstar** [scalda/FoolCraft#480]
## SUGGESTION/HELP
- [x] I would like to propose an additional mod, config change, custom recipe, etc.
- [ ] I am requesting help with installing or running the pack

## Issue Description
Hello. Today I was playing Foolcraft and noticed that for some kind of reason I get low fps in my base. But not everywhere, only looking at a certain direction. Soon I figured out that when ever I am looking at mob mashers from mob grinder utils my fps drops from 40 to 11. I removed them, looked at the same direction where my fps would drop and my fps didn't drop. (I had 8 of them at first) then I tried placing one again, after doing so my fps dropped again. I don't know how to fix this or anything. I will try using other mob killing gadgets, but just wanted to notice you about this issue if you don't already know about it.

## Reproduce Steps
I am placing a mob grinder from mob grinder utils on an enderio redstone conduit, even if the conduit is not sending a redstone signal the mob masher is still causing the fps to drop.

**Comment by Infectedstar** [comment]
Who ever is reading this, Please have a great day, you deserve it

**Comment by legodude17** [comment]
I would suggest you report it to mob_grinder_utils.

**Comment by Ravalli03** [comment]
I had an issue with this also, but it wasn't due to any mod. This may not have anything to do what you are having an issues with. I had a slime farm running over night and I always leave the area while it is running to get more mobs to spawn. When I warped back to the slime farm and found the game was unplayable, with my FPS running at 1-15 while around the mob masher. After about 20 min of fighting the FPS, It ended up being there was too many XP orbs next to the mashers and as soon as I collected them, the FPS went back to normal. I do have a hopper for the xp but it must of filled up.
